modules/admin/admin-ui/src/main/resources/web/admin/apps/content-studio/js/app/publish/ContentPublishDialog.ts

```diff
@@ -146,12 +146,12 @@ export class ContentPublishDialog extends ProgressBarDialog {
             return this.reloadPublishDependencies(childrenNotLoadedYet);
         } else {
             // apply the stash to avoid extra heavy request
-            this.lockControls();
+            this.updateSubTitleShowScheduleAndButtonCount();
             this.loadMask.show();
             setTimeout(() => {
                 this.setDependantItems(stashedItems.slice());
                 this.centerMyself();
-                this.unlockControls();
+                this.updateSubTitleShowScheduleAndButtonCount();
                 this.loadMask.hide();
```

Comment:
@edloidas  ``this.updateSubTitleShowScheduleAndButtonCount`` calls ``this.countTotal`` which is pretty heavy performance-wise (think of publishing several thousands of items). Can you explain why we need to call ``this.updateSubTitleShowScheduleAndButtonCount`` twice with 100ms delay? The number of items cannot change in this period, can it?